wiki/Community/Video_series/lazer_updates/en.md

```diff
@@ -0,0 +1,64 @@
+# lazer updates
+
+![](img/banner.png "osu!(lazer) updates newspost banner")
+
+**lazer updates** is a video series hosted by ::{ flag=US }:: [pishifat](https://osu.ppy.sh/users/3178418) and ::{ flag=AU }:: [peppy](https://osu.ppy.sh/users/2) that outlines the recent changes to the [osu!(lazer)](https://github.com/ppy/osu) client as well as its development process. The series is intended to give the general public a better look as to what features are being added or worked on in the latest and upcoming [osu!(lazer) releases](https://osu.ppy.sh/home/download).
+
+## History
+
+Documentation of osu!(lazer) updates started off in the early days as blog posts written in [peppy's personal blog](https://blog.ppy.sh/) and the [osu!next Tumblr page](https://www.tumblr.com/osunext). However, these posts were often overlooked by casual users, and in order to increase public awareness of the ongoing development, consequent updates were later announced in the [osu!(lazer)-specific changelog entries](https://osu.ppy.sh/home/changelog/lazer), [peppy's Twitter account](https://twitter.com/ppy/), and finally the "lazer updates" video series.
+
+Starting from [June 2023](https://osu.ppy.sh/home/news/2023-06-22-osulazer-updates-june-21-2023) onwards, all "lazer updates" videos are released together along with their corresponding "osu!(lazer) updates" newspost for further visibility.
+
+## List of episodes
+
+| No. | Title | Topics covered | Newspost |
```

Comment:
```suggestion
| No. | Title | Topics covered | News post |
```

Reply:
@Hiviexd e from my previous experiences `newspost` are written this way iirc...

http://osu.ppy.sh/home/search?mode=wiki_page&query=newspost

Reply:
if you look closer you'll notice that those are either old articles or one-offs, a query on "news post" shows many more recent results, in addition to other maintainers already enforcing "news post" instead of "newspost" in PRs.

https://osu.ppy.sh/home/search?mode=wiki_page&query=news%20post

Reply:
alright